extensions/adobe/experience/commerce/commerce.schema.json

```diff
@@ -0,0 +1,34 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/commerce/commerce",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Commerce",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
```

Comment:
Typo - this makes the component into a field group, so this line should only be in experienceevent-commerce